doc/reference/kernel/data_passing/message_queues.rst

```diff
@@ -202,10 +202,12 @@ in an asynchronous manner.
 .. note::
     A message queue can be used to transfer large data items, if desired.
     However, this can increase interrupt latency as interrupts are locked
-    while a data item is written or read. It is usually preferable to transfer
-    large data items by exchanging a pointer to the data item, rather than the
-    data item itself. The kernel's memory map and memory pool object types
-    can be helpful for data transfers of this sort.
+    while a data item is written or read. The time to write or read a data item
+    increases linearly with its size since the item is copied in its entirety
+    to or from the buffer in memory. For this reason, it is usually preferable
+    to transfer large data items by exchanging a pointer to the data item,
+    rather than the data item itself. The kernel's memory map and memory pool
+    object types can be helpful for data transfers of this sort.
```

Comment:
I think we should remove this:

"The kernel's memory map and memory pool
    object types can be helpful for data transfers of this sort."

We do not have dedicated memory map or memory pool objects anymore.

Reply:
Removed this and a duplicate in `doc/reference/kernel/data_passing/pipes.rst` as well :+1: